package.json

```diff
@@ -84,6 +92,7 @@
     "copy-webpack-plugin": "^4.5.2",
     "cross-env": "^5.1.4",
     "danger": "^6.0.5",
+    "del": "^5.1.0",
```

Comment:
There is only couple of places left where `rimraf` is used. Shouldn't we replace them with `del` and get rid of `rimraf` completely?